x-pack/legacy/plugins/siem/public/pages/hosts/hosts.tsx

```diff
@@ -187,22 +187,22 @@ const HostsComponent = pure<HostsComponentProps>(({ filterQuery, setAbsoluteRang
                         totalCount,
                         loading,
                         pageInfo,
-                        loadMore,
+                        loadPage,
                         id,
                         inspect,
                         refetch,
                       }) => (
                         <UncommonProcessTableManage
+                          data={uncommonProcesses}
+                          fakeTotalCount={getOr(50, 'fakeTotalCount', pageInfo)}
```

Comment:
I see 50 being used in a lot of areas with this PR, suggestion but optional is to pull it out into a single constant?

Reply:
it doesnt need to be 50, its just for tests